src/protocol/http/org/apache/jmeter/protocol/http/sampler/HTTPSamplerBase.java

```diff
@@ -926,7 +966,44 @@ public CookieManager getCookieManager() {
     private void setCacheManagerProperty(CacheManager value) {
         setProperty(new TestElementProperty(CACHE_MANAGER, value));
     }
+    private void setKeystoreConfigProperty(KeystoreConfig value) {
+        setProperty(new TestElementProperty(KEYSTORE_CONFIG, value));
+    }
+
+    public void setKeystoreConfig(KeystoreConfig value) {
+        KeystoreConfig mgr = getKeystoreConfig();
+        if (mgr != null) {
+            if (log.isWarnEnabled()) {
+                log.warn("Existing CookieManager {} superseded by {}", mgr.getName(), value.getName());
```

Comment:
La log n'est pas bonne, est-elle justifiée ?

Reply:
I have modified it, sorry I should pay attention to it